lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -719,14 +719,20 @@ def create_or_update_elb_listeners(connection, module, elb):
     for listener_to_add in listeners_to_add:
         try:
             listener_to_add['LoadBalancerArn'] = elb['LoadBalancerArn']
-            connection.create_listener(**listener_to_add)
+            # Rules is not a valid parameter for create_listener
```

Comment:
Good spot for lines 691 & 692 - I'll fix this.

Hope I'm not being dim, but what does "so the invocation gets shown correctly" refer to? The change you suggest for line 744 seems harmless but I don't understand the reason for it at the moment.

Thanks!